src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -0,0 +1,171 @@
+= Testing templates
+:page-slug: testing-templates
+:page-description: Guide to testing Koji templates.
+
+There are many powerful testing tools built into the Koji platform.
+Typically, you will begin testing in the Koji editor.
+Later, you'll want to publish the template as *Unlisted*, so you can test it in a production environment before giving access to general users.
+Some of the most common testing methods are listed below.
+
+== Testing in the Koji editor
+
+The Koji editor provides various ways to preview a template's behavior.
+
+=== Previewing the Koji
```

Comment:
"Koji" vs. "template" - [this style note](https://github.com/madewithkoji/docs/pull/109#discussion_r536420469) still needs to be addressed throughout the page.